content/doc/book/managing/plugins.adoc

```diff
@@ -387,3 +387,43 @@ image::managing/hover-for-implied-dependencies.png["Hover over the uninstall but
 
 The implied dependency can be removed by releasing a new version of the plugin that depends on a newer minimum Jenkins core. 
 The link:/doc/developer/tutorial-improve/[Improve a plugin tutorial] provides steps that can assist plugin developers as they update the plugin to link:/doc/developer/tutorial-improve/update-base-jenkins-version/[depend on a newer minimum Jenkins core].
+
+[[plugin-health-score]]
+== What is the health score?
+
+There is a metric available next to the plugins in the Plugin Manager called "Health Score".
+
+image::managing/plugin-manager-available-health-score.png["Plugin Health Score in the Plugin Manager available listing", role=center]
+
+You can find the same metric in the available update and the installed page of the plugin manager.
+
+NOTE: The score value is always based on the latest available data.
+This means that there is no score of a specific release of a plugin.
+What is available on the installed and updates page of the plugin manager does not stand for the version already installed.
+
+This metric is coming from link:https://github.com/jenkins-infra/plugin-health-scoring[Plugin Health Scoring].
+
+This service is gathering data from each plugin.
+
+The data can be related to the source code of the plugin, the content of the plugin repository or external metrics about the plugin such as number of open pull requests, etc.
+
+That data is then used to evaluate the status of the project.
+All plugins are evaluated the same way, meaning there is a fair way to compare plugins.
+
+Only the score is shown, but the details behind this value are available on the link:https://plugins.jenkins.io[plugins site], in the health score section of each plugin:
+
+image::managing/plugin-site-health-score.png["Plugin health score on the plugins site", role=center]
+
+It is also possible to find the details directly on the link:https://plugin-health.jenkins.io[Plugin Health Scoring] service.
+To access it, you need to append the plugin name to the score URL.
+For example, the health score for `mailer` plugin is `https://plugin-health.jenkins.io/scores/mailer`.
+
+Each section of the score can have one to multiple element evaluated.
```

Comment:
```suggestion
Each section of the score can have one or multiple elements evaluated.
```
Suggestion to adjust the grammar here so that elements matches multiple (plural).